pallets/treasury/src/lib.rs

```diff
@@ -235,6 +235,7 @@ impl<T: Config> OnUnbalanced<NegativeImbalanceOf<T>> for Module<T> {
         let numeric_amount = amount.peek();
 
         let _ = T::Currency::resolve_creating(&Self::account_id(), amount);
+        // TODO: Should we use a fixed value here?
         let current_did = Context::current_identity::<Identity<T>>().unwrap_or_default();
         Self::deposit_event(RawEvent::TreasuryReimbursement(current_did, numeric_amount));
```

Comment:
Is the caller's DID really important in this event?  The `Self::account_id()` will always be the treasury account id.  `current_identity()` should return the identity that made the transaction or is paying for the transaction.  Transaction fees no longer pay into the treasury, so this is only used for Slashing.  The slashing might not set current identity, so this would just use the default DID (all zeroes).

It has been over 400 days since the last time this event has been emitted.

Reply:
Should just hardcode to `GC_DID` for now.